benchs/bench-cpu-mem/Bench/Scripts.hs

```diff
@@ -28,3 +34,55 @@ plonkVerifierScript paramsSetup redeemerProof ctx =
        `unsafeApplyCode` liftCodeDef paramsSetup
        `unsafeApplyCode` liftCodeDef redeemerProof
        `unsafeApplyCode` liftCodeDef ctx
+
+verifyPlonkScript :: SetupBytes -> InputBytes -> ProofBytes -> UPLC.Program UPLC.NamedDeBruijn DefaultUni DefaultFun ()
```

Comment:
This one seems redundant since we have `compiledVerifyPlonk` just below.

Reply:
I'm not sure -- they have different type signatures, so I can't just replace `verifyPlonkScript` with `compiledVerifyPlonk` in e.g. `printStatistics` function. Or am I missing something?